sass/directives/_flyout.scss

```diff
@@ -87,3 +87,117 @@
     }
   }
 }
+
+// Use the mobile-subnav mixin for the subnav bars that appear on our subscription / product pages
+// Displays a single link with a flyout
+// Structure of markup should be:
+@mixin mobile-subnav-flyout() {
+  width: auto; 
+  display: none;
+  position: relative;
+  cursor: pointer;
+
+  @media only screen and (max-width: $small-screen-max) {
+    line-height: 45px;
+    display: block;
+    height: $product_subnav--height;
+    margin-left: -$container-edge-padding;
+    margin-right: -$container-edge-padding;
+  }
+  
+  &:hover {
+    ul {
+      display: block;
+    }
+
+    .mobile-subnav__top-link:after {
+      content: '';
+    }
+  }
+
+  ul {
+    display: none;
+    background-color: white;
+    border-bottom: $base-border;
+    position: absolute !important;
```

Comment:
this seems to still work/apply without the `!important` 

![image](https://user-images.githubusercontent.com/2359538/27444417-eac6bcc0-573b-11e7-919b-920d4635aa07.png)